.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: E4E Pipeline
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Audit
+        id: audit
+        continue-on-error: true
+        run: yarn audit
+
+      - name: Lint
+        run: yarn workspace explorer-for-endevor lint
+
+      - name: Run tests
+        uses: GabrielBB/xvfb-action@v1.0
+        with:
+          run: yarn workspace explorer-for-endevor test
+
+      - name: Build package
+        if: steps.audit.outputs.exit_code < 7
+        run: yarn workspace explorer-for-endevor build
+
+      - name: Package VSIX & Upload Artifact
+        if: steps.audit.outputs.exit_code < 7
+        run: yarn workspace explorer-for-endevor package
+      - uses: actions/upload-artifact@v2
+        with:
+          path: packages/explorer-for-endevor/vsce/*.vsix
```

Comment:
Minor: can I ask you to specify name of the artefact here as well? Because now it is just `artefact`. Maybe `explorer-for-endevor.vsix` will be better?

Reply:
Not sure if this is needed now but I would prefer this format `explorer-for-endevor-0.13.0.vsix`